### ISSUE TITLE
Docker fails to download podcasts

### ISSUE THREAD

**Issue by joshaconnor** [rmcrackan/Libation#572]
**Describe the bug**
The docker version gets hung on downloading podcasts. I am unable to download even the first episode to see if it works for more episodes. 

**To Reproduce**

1. Install Docker version
2. Have at least one book and one podcast in your library (the specific hung podcast for me is WWW 3 from Worlds Beyond Number)
3. Look at the logs. 

**Expected behavior**
Podcasts should download, then decrypt(?), then move on. 

**Platform**

Linux, Ubuntu 22.04
Docker

**Relevant area of Log Files**
```
 {}2023-04-05 02:34:04.411 +00:00 [INF] (at Dinah.Core.IO.MultiTextWriter.WriteLine(System.String)) DownloadDecryptBook Begin: 03/28/2023 [B09ZY37YQH] WWW #3: The Charter
 {}2023-04-05 02:34:04.446 +00:00 [INF] (at AudibleUtilities.ApiExtended+<CreateAsync>d__9.MoveNext()) {REDACTED}
 {}2023-04-05 02:34:05.078 +00:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) Begin step 'Download Unencrypted Audiobook'
 {}2023-04-05 02:34:05.088 +00:00 [INF] (at Dinah.Core.StepRunner.AsyncBaseStep+<RunAsync>d__2.MoveNext()) Begin step 'Step 1: Download Audiobook'
 {}
```
```
2023-04-05 02:34:01: Liberating books
DownloadDecryptBook Begin: 03/28/2023 [B09ZY37YQH] WWW #3: The Charter
```

**Extra Info**
[Specific podcast with issue](https://www.audible.com/pd/Worlds-Beyond-Number-Podcast/B0BRW4Z53D?action_code=ASSGB149080119000H&share_location=pdp)

**Comment by joshaconnor** [comment]
Side note, the podcast episode will download on a different machine using the macOS install package UI. I don't know about on the same machine with the non-docker flavor.

**Comment by Mbucari** [comment]
Why don't you use the Deb package?

**Comment by joshaconnor** [comment]
I mean, the sarcastic answer is why is there a docker option if its not meant to be used. I suspect that you would prefer a less condescending response though. 

The more accurate answer is that I have several self-hosted apps (presently 17 with plans to expand to 25ish) on my server and the nightmare of trying to run everything directly on the machine would send me over the edge. The server I have running is meant to be automated - or as near as possible - in its tasks, and having the system be containerized makes that significantly easier. The docker compose file I have is meant to be added to a brand new install of Ubuntu and have all my apps running in 10 minutes or less with almost no additional work, so I can teardown and rebuild regularly. 

Philosophically, running a containerized app presents a lot of advantages including security, customizability, portability, scalability, ease of use, etc. 

Lastly, everything else I have is running in docker and I like consistency.

**Comment by Mbucari** [comment]
For some more context, the docker image was made as a one-off contribution by a volunteer. Neither of the two maintainers know how to do docker. I have no intention of learning, and @rmcrackan doesn't have the time. The docker image is available because it may be useful in whatever capacity it happens to be working, but it is not officially supported.

**Comment by rmcrackan** [comment]
@Mbucari Was that only @pixil98? or did someone else also help? Maybe they'd be willing to take a look.

**Comment by rmcrackan** [comment]
@Mbucari I just added a disclaimer to the docker page. Guess I missed adding it before. Oops

**Comment by joshaconnor** [comment]
I believe @wtanksleyjr is active in the docker version, or their comments on other docker issues would suggest so.

**Comment by wtanksleyjr** [comment]
I tried to use it, but eventually gave up.

I think what might be going on is that the Docker version is trying to do
too much; it _should_ just host libation-cli to provide commands that can
be run outside of Docker (while hosting Libation's required environment and
providing mappings to allow the required DB and config volumes), but
instead it tries to cron a script that regularly checks (and stuff like
that).

I don't know what I'm doing, unfortunately. I was able to figure out how to
run it as non-root, which is more Docker knowledge than I'd had before.

-Wm
